### PR TITLE
[textanalytics] remove extractive text summarization

### PR DESCRIPTION
**Pull request by kristapratico**
Part of https://github.com/Azure/azure-sdk-for-python/issues/24769

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py [hunk 1]
```diff
@@ -1777,10 +1775,6 @@ class ActionPointerKind(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     EXTRACT_KEY_PHRASES = "keyPhraseExtractionTasks"
     RECOGNIZE_LINKED_ENTITIES = "entityLinkingTasks"
     ANALYZE_SENTIMENT = "sentimentAnalysisTasks"
-    EXTRACT_SUMMARY = "extractiveSummarizationTasks"
-    RECOGNIZE_CUSTOM_ENTITIES = "customEntityRecognitionTasks"
-    SINGLE_CATEGORY_CLASSIFY = "customSingleClassificationTasks"
-    MULTI_CATEGORY_CLASSIFY = "customMultiClassificationTasks"
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:1783]: Removing stale code, this is only used in v3.1

Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:1783]: Removing stale code, this is only used in v3.1

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py [hunk 2]
```diff
@@ -1278,22 +1273,6 @@ def get_result_from_cont_token(initial_response, pipeline_response):
...
-                    if _determine_action_type(a) == _AnalyzeActionsType.RECOGNIZE_CUSTOM_ENTITIES
-                ],
-                custom_single_classification_tasks=[
-                    a for a in generated_tasks
-                    if _determine_action_type(a) == _AnalyzeActionsType.SINGLE_CATEGORY_CLASSIFY
-                ],
-                custom_multi_classification_tasks=[
-                    a for a in generated_tasks
-                    if _determine_action_type(a) == _AnalyzeActionsType.MULTI_CATEGORY_CLASSIFY
-                ],
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py:1296]: removing more stale code

Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py:1296]: removing more stale code

**Review by iscai-msft — APPROVED** [review]

**Review by deyaaeldeen — APPROVED** [review]

**Pull request merged**